src/aap_eda/settings/default.py

```diff
@@ -765,3 +765,8 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 MAX_PG_NOTIFY_MESSAGE_SIZE = int(
     settings.get("MAX_PG_NOTIFY_MESSAGE_SIZE", 6144)
 )
+
+AUTOMATION_ANALYTICS_URL = settings.get("AUTOMATION_ANALYTICS_URL", "")
+INSIGHTS_CERT_PATH = settings.get(
+    "INSIGHTS_CERT_PATH", "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
```

Comment:
The default should be an empty string. Otherwise, it will complain no certificate file is found when the default path is invalid.